enos/ansible/plugins/callback/influxdb_events.py

```diff
@@ -30,107 +26,129 @@ class CallbackModule(CallbackBase):
     CALLBACK_NAME = 'influxdb_events'
     CALLBACK_NEEDS_WHITELIST = False
 
-    def __init__(self):
 
+    def __init__(self):
         super(CallbackModule, self).__init__()
 
+        self.timer = None
         self.events = []
-        self.playbook = None
-        self.playbook_name = None
-
-    def report_event(self, event_type, fields):
-        """
-        Add a new event in the list
-        """
-        tags={}
-        tags['type'] = event_type
-        tags['user']= tags.get('user', self.username)
-        tags['hostname'] = tags.get('hostname', self.hostname)
+        self.extra_vars = None
+        self.path = self._original_path
+        self.timer = None
+        self.username = pwd.getpwuid(os.getuid()).pw_name
+
+
+    def report_event(self, fields):
+        """Add a new event in the list"""
+
+        # Add Kolla-related information in tags
+        fields['tags'] = '%s %s %s %s' % (fields['tags'],
+                self.extra_vars['kolla_ref'],
+                self.extra_vars['kolla']['kolla_base_distro'],
+                self.extra_vars['kolla']['kolla_install_type'])
+
         current_time = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')
-        event = [
-            {
-                'time': current_time,
-                'measurement': 'events',
-                'tags': tags,
-                'fields': fields
-            }
-        ]
+
+        event = {
+            'time': current_time,
+            'measurement': 'events',
+            'fields': fields
+        }
+
         self.events.append(event)
 
+
     def v2_playbook_on_start(self, playbook):
-        """
-        Logs each playbook start
-        """
-        self.username = pwd.getpwuid(os.getuid()).pw_name
-        self.hostname = socket.gethostname()
-        self.playbook = playbook
-        self.playbook_name = os.path.basename(playbook._file_name)
-        self.playbook_basedir = playbook._basedir
+        """Log each starting playbook"""
+
+        playbook_name = os.path.basename(playbook._file_name)
+
+        variable_manager = playbook._entries[0]._variable_manager
+        self.extra_vars = variable_manager.extra_vars
+
+        # Check if `enable_monitoring` is set
+        if not self.extra_vars.get('enable_monitoring', True):
+            # Disable the plugin if InfluxDB is unreachable
+            self.disabled = True
```

Comment:
Is `self.disabled` a pre-defined attribute of ansible Callback plugins ? Otherwise It seems unused in this file.